corehq/apps/translations/app_translations.py

```diff
@@ -283,20 +283,22 @@ def get_bulk_multimedia_sheet_rows(lang, app):
                                                      [module.audio_by_language(lang)]))
 
         # Detail case properties, etc.
-        for row in get_module_rows([lang], module):
-            rows.append(prefix + list(row))
+        if not isinstance(module, ReportModule):
```

Comment:
Similarly to other comment, this check should go inside `get_module_rows` rather than it being the caller's responsibility to be knowledgeable about the different types of modules.